go-packages/meep-net-char-mgr/net-char-mgr_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2019  InterDigital Communications, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package netchar
+
+import (
+	"fmt"
+	"testing"
+
+	log "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-logger"
+)
+
+const netCharMgrRedisAddr string = "localhost:30380"
+
+// // Callback function to update a specific filter rule
+// func updateFilterRule(string, string, float64) {
+
+// }
+
+// // Callback function to apply filter rule updates
+// func applyFilterRule() {
+
+// }
```

Comment:
needed?

Reply:
Eventually, yes. The callback functions are currently not unit tested but they should.